packages/core/src/avatars.json

```diff
@@ -2,6 +2,7 @@
 	"blue": "https://cdn.discordapp.com/embed/avatars/0.png",
 	"gray": "https://cdn.discordapp.com/embed/avatars/1.png",
 	"green": "https://cdn.discordapp.com/embed/avatars/2.png",
-	"orange": "https://cdn.discordapp.com/embed/avatars/3.png",
-	"red": "https://cdn.discordapp.com/embed/avatars/4.png"
+	"yellow": "https://cdn.discordapp.com/embed/avatars/3.png",
```

Comment:
```suggestion
	"orange": "https://cdn.discordapp.com/embed/avatars/3.png",
```
Not gonna change this to yellow because it seems more orange than anything to me. 😅 There's nothing official indicating whether it is orange or yellow, so I'll stick with orange.

Reply:
@Danktuary 
Source: [Click here](https://discord.com/branding 'From Discord\'s website.')
![image](https://user-images.githubusercontent.com/45759535/126975506-c65e8d23-0625-4615-ad72-21535936e500.png)